bundle/next/eclipse-che/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -934,21 +942,21 @@ spec:
                       - name: OPERATOR_NAME
                         value: che-operator
                       - name: CHE_VERSION
-                        value: next
+                        value: 7.82.0
```

Comment:
Please revert all changes related to images

Reply:
I've removed the rest of the changes to images. Thank you for the review!